packages/core/src/exchange/listToken.ts

```diff
@@ -69,11 +69,26 @@ export type Listing = {
    * @type Date
    */
   expiredAt?: Date
+
+  /**
+   * The metadata associated to the bid
+   * @type JSON
+   */
+  metadata?: JSON
```

Comment:
JSON type is not working
```suggestion
  metadata?: unknown
```

Reply:
unknown is anyway better than JSON 👍

Reply:
done